liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -164,10 +186,21 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
 	// exception header line
-	errorColored() << _msg.severity;
-	if (m_withErrorIds && _msg.errorId.has_value())
-		errorColored() << " (" << _msg.errorId.value().error << ")";
-	messageColored() << ": " << _msg.primary.message << '\n';
+	if (_msg.errorId.has_value())
+	{
```

Comment:
This does not have a corresponding closing brace. This is a syntax error.

Reply:
Done 👍